scripts/remove_operator.py

```diff
@@ -32,3 +32,5 @@ def delete_resource(api, resource):
     delete_resource("operatorversion.kudo.dev", version)
 
 delete_resource("operator.kudo.dev", "spark")
+delete_resource("crd", "sparkapplications.sparkoperator.k8s.io")
```

Comment:
Not in scope of this PR but since we're using python, maybe we want to add `retrying` for these deletes in case of network issues